src/main/java/bo/usfx/jhoselineteran/TotalPoints.java

```diff
@@ -0,0 +1,65 @@
+package bo.usfx.jhoselineteran;
+
+import java.util.logging.Logger;
+
+/**
+ * The TotalPoints class provides a method to
+ * calculate the total points earned by a team in a series of games.
+ */
+public final class TotalPoints {
+  /**
+   * Logger instance for logging information.
+   */
+  private static final Logger LOGGER =
+          Logger.getLogger(TotalPoints.class.getName());
+  /**
+   * Constant representing the points awarded for a win in a game.
+   */
+
+  private static final int WIN_POINTS = 3;
+
+  private TotalPoints() {
+    // Constructor privado para ocultar el constructor predeterminado
+  }
+
+  /**
+   * Calculates the total points earned by
+   * a team based on the results of a series of games.
+   *
+   * @param games An array of strings representing
+   *              the scores of each game in the format "X:Y".
+   * @return The total points earned by the team.
+   */
+
+  public static int points(final String[] games) {
+    int totalPoints = 0;
+
+    for (final String game : games) {
+      final String[] scores = game.split(":");
+      final int ourScore = Integer.parseInt(scores[0]);
+      final int opponentScore = Integer.parseInt(scores[1]);
+
+      if (ourScore > opponentScore) {
+        totalPoints += WIN_POINTS;  // Win
+      } else if (ourScore == opponentScore) {
+        totalPoints += 1;  // Tie
+      }
+      // No need to add points for a loss (0 points)
+    }
+    return totalPoints;
+  }
+  /**
+   * The main method demonstrates the usage of the TotalPoints
+   * class by calculating points earned in a predefined set of games.
+   *
+   * @param args Command line arguments (not used in this context).
+   */
+
+  public static void main(final String[] args) {
```

Comment:
Remove this void main method. It's not necessary, currently its reducing coverage.